doc-content/enterprise-only/openshift/hacep-server-maven-proc.adoc

```diff
@@ -0,0 +1,142 @@
+[id='hacep-server-maven-proc']
+= Implementing the HA CEP server with a Maven repository for updating the KJAR service
+
+You can implement the HA CEP server with a separate Maven repository and place your KJAR service in the repository. In this case, you can update the KJAR service at any time by updating it in the Maven repository and then updating the group, artifact, and version (GAV) values in an environment variable of the server pod.
+
+You must prepare the source, build it, and then deploy it on {OPENSHIFT}. 
```

Comment:
"You must prepare the source, build it, then add the following env var in the yaml, and then then deploy it on {OPENSHIFT}. "

```
name: UPDATABLEKJAR
      value: "true"
```

I think must me remarked that this env var is the key to enable the update at runtime